src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -65,42 +81,58 @@ const TabbedLayout: FunctionComponent<TabbedLayoutProps> = ({
         }
     };
 
+    const toggleCollapse = () => setCollapsed(!collapsed);
+
     return (
         <Box className={ classes.root } display="flex" height="100vh">
             <OrganizeSidebar />
             <Box display="flex" flexDirection="column" height="100vh" overflow="auto" position="relative" width={ 1 }>
                 <Box display={ fixedHeight ? 'flex' :'block' } flexDirection="column" height={ fixedHeight ? 1 : 'auto' }>
                     <Box component="header" flexGrow={ 0 } flexShrink={ 0 }>
-                        <Box mb={ 2 } pt={ 3 } px={ 3 }>
-                            <BreadcrumbTrail/>
-                            { /* Title, subtitle, and action buttons */ }
-                            <Box className={ classes.titleGrid } mt={ 2 }>
-                                <Box overflow="hidden">
-                                    <Typography className={ classes.title } component="h1" data-testid="page-title" noWrap variant="h3">
-                                        { title }
-                                    </Typography>
-                                    <Typography component="h2" variant="h5">
-                                        { subtitle }
-                                    </Typography>
-                                </Box>
-                                <Box>
-                                    { actionButtons }
+                        <Box mb={ collapsed ? 1 : 2 } pt={ 3 } px={ 3 }>
+                            <Box display="flex" flexDirection="row" justifyContent="space-between">
+                                <BreadcrumbTrail highlight={ collapsed } />
+                                <Box className="collapse-button">
+                                    <Button fullWidth onClick={ toggleCollapse } size="small"
+                                        startIcon={  <ArrowUpward /> }>
+                                        { intl.formatMessage({
+                                            id: `layout.organize.header.collapseButton.${collapsed ? 'expand' : 'collapse'}`,
+                                        }) }
```

Comment:
Ok I am sorry to be super pedantic but for one there is the component `<FormattedMessage />` which handles this which we tend to use when putting localised text in JSX.

Reply:
2 instances of useIntl hook replaced with ´<FormattedMessage />´  😉